### PR TITLE
PMM-3313 Add PostgreSQL to PMM Server

### PR DESCRIPTION
**Pull request by adivinho**

### CODE REVIEW

File: supervisord.conf [hunk 1]
```diff
@@ -27,6 +27,17 @@ stdout_logfile = /var/log/mysql.log
 stderr_logfile = /var/log/mysql.log
 autorestart = true
 
+[program:postgresql]
+priority = 1
+command =
+    /usr/pgsql-10/bin/postgres
+        -D /srv/postgres
+        -r /srv/logs/postgres.log
```
Comment (AlekSi) [inline supervisord.conf]: That will not work. First, `-r` works only in single-user mode. Second, we want postgres to log to stdout/stderr and let supervisor redirect those streams to file (that's what lines below are doing).

File: supervisord.conf [hunk 2]
```diff
@@ -27,6 +27,17 @@ stdout_logfile = /var/log/mysql.log
...
+stdout_logfile = /srv/logs/postgres.log
+stderr_logfile = /srv/logs/postgres.log
+user=postgres
+autorestart=true
```
Comment (AlekSi) [inline supervisord.conf]: > To terminate the postgres server normally, the signals SIGTERM, SIGINT, or SIGQUIT can be used. The first will wait for all clients to terminate before quitting, the second will forcefully disconnect all clients, and the third will quit immediately without proper shutdown, resulting in a recovery run during restart.

We need `SIGINT`, but supervisord uses `SIGTERM` by default. We need to change that option.

Comment (adivinho) [reply supervisord.conf]: The stop signal has been changed.
Here are logs of my tests
---
2018-12-21 09:54:24,806 DEBG killing postgresql (pid 20) with signal SIGINT
2018-12-21 09:54:24,834 INFO stopped: postgresql (exit status 0)
---

File: supervisord.conf (hunk 1 above)
Comment (adivinho) [reply supervisord.conf]: The parameter has been removed

**Pull request merged**